README.md

```diff
@@ -125,18 +125,20 @@ checkpoint and run with
 --restore_prior="path/to/checkpoint" --lr_use_linear_decay --lr_start_linear_decay={already_trained_steps} --lr_decay={decay_steps_as_needed}
 ```
 
-### Reuse pre-trained VQ-VAE and retrain top level prior on new dataset.
+### Reuse pre-trained VQ-VAE and train top-level prior on new dataset from scratch.
 #### Train without labels
 Our pre-trained VQ-VAE can produce compressed codes for a wide variety of genres of music, and the pre-trained upsamplers 
 can upsample them back to audio that sound very similar to the original audio.
 To re-use these for a new dataset of your choice, you can retrain just the top-level  
 
-To retrain top-level on a new dataset, run
+To train top-level on a new dataset, run
 ```
 mpiexec -n {ngpus} python jukebox/train.py --hps=vqvae,small_prior,all_fp16,cpu_ema --name=pretrained_vqvae_small_prior \
 --sample_length=1048576 --bs=4 --aug_shift --aug_blend --audio_files_dir={audio_files_dir} \
---labels=False --train --test --prior --levels=3 --level=2 --weight_decay=0.01 --save_iters=1000
+--labels=False --train --test --prior --levels=3 --level=2 --weight_decay=0.01 --save_iters=1000 \
+--lr_use_linear_decay --lr_decay={decay_steps_as_needed}
 ```
```

Comment:
Lets remove linear decay here, since I added a separate step for it in the comment in line 143

Reply:
ah okay